src/modules/searchMoviesReducer.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+import api from 'utils/Tmdb.js';
+
+const initialState = {
+  data: [],
+  isLoaded: false,
+  searchMovies: [],
+};
+
+const MOVIE_LOADING = 'MOVIE_LOADING';
+const MOVIE_LOADED = 'MOVIE_LOADED';
+
+export default function reducer(state: Object = initialState, action: Object) {

```

Comment:
Pls, leave `TODO` that we need some concrete types for the state and for actions.